02-peer-protocol.md

```diff
@@ -953,6 +953,9 @@ fee changes).
   to BIP69 lexicographic ordering of the commitment transaction.
   - if it has not recently received a message from the remote node:
       - SHOULD use `ping` and await the reply `pong` before sending `commitment_signed`.
+  - after sending a `commitment_signed` message:
+      - MUST NOT process any `update` message from the other party before receiving the
+        `revoke_and_ack` that corresponds to the `commitment_signed` sent.
```

Comment:
This should also clarify what happens if the counterparty continues to send updates. Should it queue them up? Should it fail the channel?

Reply:
I propose that further `update` messages should be simply ignored. I've updated the text accordingly.